bundle/config/workspace.go

```diff
@@ -89,10 +94,10 @@ func (w *Workspace) Config() *config.Config {
 		// Having client-side timeouts that kill the deployment seems counter-productive. We should just keep on
 		// trying and the user should be the one interrupting it if they decide so.
 		// Default is 30s
-		HTTPTimeoutSeconds: 90,
+		HTTPTimeoutSeconds: WorkspaceClientHTTPTimeoutSeconds,
```

Comment:
I'm not sure we should blindly reuse timeouts. If there is an issue with certain API and we have 20 tests failing and they are not concurrent then we'll have to wait a long time to get the results.

I'd keep the timeouts independent and increase HTTP timeout slightly for integration tests but keep retry timeout the same.